storage/store.go

```diff
@@ -1164,7 +1166,8 @@ func (s *Store) ExecuteCmd(ctx context.Context, call client.Call) error {
 		// intent independently, so we can't do it in Range.executeCmd.
 		if wiErr, ok := err.(*proto.WriteIntentError); ok {
 			// If inconsistent, return results and resolve asynchronously.
-			if header.ReadConsistency == proto.INCONSISTENT && s.stopper.StartTask() {
+			if header.ReadConsistency == proto.INCONSISTENT &&
+				attempts == 1 && s.stopper.StartTask() {
```

Comment:
no, you're the one who got it right. I didn't see that code path but it makes sense. The retries in the put_test stem come from the DistSender anyways. I'll close this.